### PR TITLE
Grammar edits on single player scenario 20 - supremacy

### PR DESCRIPTION
**Pull request by alwaysnicki**

### CODE REVIEW

File: singleplayer/scenarios/scenario022.lua [hunk 1]
```diff
@@ -1,652 +1,652 @@
...
+	index			= 22, --  integer, sort order, MUST BE EQUAL TO FILENAME NUMBER
+	scenarioid		= "Begin06", -- no spaces, lowercase, this will be used to save the score and can be used gadget side
+    version         = "1", -- increment this to reset the score when changing a mission, as scores are keyed by (scenarioid,version,difficulty)
+	title			= "Supremacy", -- can be anything
+	author			= "wilkubyk", -- your name here
+	isnew = true,
+	imagepath		= "scenario022.jpg", -- placed next to lua file, should be 3:1 ratio banner style
+	imageflavor		= "River Assault", -- This text will be drawn over image
+    summary         = [[After a ferocious battle, a Cortex Commander decided to join Armada forces. Unfortunately, other Cortex commanders discovered the intent of the treacherous plan and sent a mighty force to prevent it.]],
+	briefing 		= [[You will start with some Lazarus (Resurrection and Repair Bots), that can resurrect units from their wrecks. Your Only Hope is to hide your position and build a mighty army to crush all opposing forces.
```
Comment (sprunk) [inline singleplayer/scenarios/scenario022.lua:11]: only still uppercase?

File: singleplayer/scenarios/scenario022.lua [hunk 2]
```diff
@@ -1,652 +1,652 @@
...
+       
+	!!! DO NOT UNDERESTIMATE YOUR ENEMY !!!
+      
+  While you will be fighting for your life , don't forget to fortify your Ally too, since WE can't lose the Cortex Command Tower.
```
Comment (sprunk) [inline singleplayer/scenarios/scenario022.lua:15]: space before comma, randomly uppercase WE

File: singleplayer/scenarios/scenario022.lua [hunk 3]
```diff
@@ -1,652 +1,652 @@
...
+	imageflavor		= "River Assault", -- This text will be drawn over image
+    summary         = [[After a ferocious battle, a Cortex Commander decided to join Armada forces. Unfortunately, other Cortex commanders discovered the intent of the treacherous plan and sent a mighty force to prevent it.]],
+	briefing 		= [[You will start with some Lazarus (Resurrection and Repair Bots), that can resurrect units from their wrecks. Your Only Hope is to hide your position and build a mighty army to crush all opposing forces.
+       
+	!!! DO NOT UNDERESTIMATE YOUR ENEMY !!!
+      
+  While you will be fighting for your life , don't forget to fortify your Ally too, since WE can't lose the Cortex Command Tower. 
+  
+Reinforcements:
+ - You and your Ally will receive the first contingency around the 9th minute from the begin of mission.
```
Comment (sprunk) [inline singleplayer/scenarios/scenario022.lua:18]: beginning

File: singleplayer/scenarios/scenario022.lua [hunk 4]
```diff
@@ -1,652 +1,652 @@
...
+ - The enemy will be getting Reinforcements every 8 and 20 minutes.
+    
+Tips:
+ - Sharing is Caring! When you share limited units & buildings with your Ally this will strengthen economy and combat capabilities of both of you.
```
Comment (sprunk) [inline singleplayer/scenarios/scenario022.lua:22]: strengthen the? but also this sentence isn't even true

**Comment by Beherith** [comment]
@alwaysnicki , these changes should mainly go in the BYAR-Chobby repo, as those contain the player-facing text. They are only copied into this repo for redundancy.

File: singleplayer/scenarios/scenario022.lua (hunk 4 above)
Comment (alwaysnicki) [reply singleplayer/scenarios/scenario022.lua:22]: I didn't know that, I didn't write the scenario but I can remove it

File: singleplayer/scenarios/scenario022.lua (hunk 3 above)
Comment (alwaysnicki) [reply singleplayer/scenarios/scenario022.lua:18]: nice catch!

File: singleplayer/scenarios/scenario022.lua (hunk 2 above)
Comment (alwaysnicki) [reply singleplayer/scenarios/scenario022.lua:15]: not sure why that was the case, didn't want to mess with artistic license

File: singleplayer/scenarios/scenario022.lua (hunk 1 above)
Comment (alwaysnicki) [reply singleplayer/scenarios/scenario022.lua:11]: I thought it was a star wars reference but didn't want to make it too obvious adding the My

**Comment by alwaysnicki** [comment]
ok I'll move them to the other repo with the changes you suggested.

**Pull request closed without merging**